test.py

```diff
@@ -37,25 +37,34 @@
 import treelib
```

Comment:
I'm looking at the second commit and it looks like it does 5-10 things at once that should be in separate commits. I have no idea why some of these changes are needed, for example why we remove `skip_in_release` out of nowhere, and why we suddenly need `test_config.yaml` alongside `scylla.yaml`.

Maybe all of these changes need to be together and they somehow make sense. I'll try to review again, but it possible, please try to make to make this commit easier to review.